ci/Jenkinsfile

```
@@ -0,0 +1,16 @@
+/*
+** Pipeline code.
+*/
+
+stage('Dependencies containers creation') {
+    parallel 'debian 11 dependencies': {
+        node {
+            dir('centreon-widget-grid-map-debian11') {
+                checkout scm
+                dir ('ci/docker') {
+                    sh 'docker build --no-cache . -f Dockerfile.widget-debian11-dependencies -t registry.centreon.com/centreon-widget-debian11-dependencies:22.10'
```

```suggestion
                    sh 'docker build --no-cache . -f Dockerfile.widget-debian11-dependencies -t registry.centreon.com/centreon-widget-debian11-dependencies:22.10'
```
this shouldn't be 22.04 instead of 22.10